tests/io/test_particle_collection.py

```diff
@@ -70,19 +74,17 @@ def test_yaml_to_xml():
     io.write(yaml_particle_collection, xml_file)
     xml_particle_collection = io.load_particle_collection(xml_file)
     assert xml_particle_collection == yaml_particle_collection
-    dummy_particle = Particle(name="0", pid=0, charge=0, spin=0, mass=0)
+    dummy_particle = Particle(
+        name="0", pid=0, mass=0, state=QuantumState(charge=0, spin=0)
```

Comment:
we could also make default values for charge (=0) and possibly spin (that might get a bit tricky with the optional projection though)

Reply:
To be honest, I don't understand how this can pass `mypy`. I mean, `spin`, should be `Spin`, not `int`...

Reply:
So the typing error is fixed now (6e756bb8cde5b9e359c39fe852d29c3fe125424e), but this doesn't settle the argument yet. There is indeed something to say for providing default values for all attributes of `QuantumState` (both from a user point of view and a physics point of view).

Reply:
@spflueger Check https://github.com/ComPWA/expertsystem/tree/WIP/dataclass-attempt. It's not really functional, but illustrates the idea. The big problem remains the default values, see https://stackoverflow.com/a/51595616.

Reply:
Okay, all has been addressed in 4140b87f26b3c06d9080fa0332f8ee4a98a67330 (thanks @spflueger)